scene/gui/texture_progress_bar.cpp

```diff
@@ -710,8 +722,13 @@ void TextureProgressBar::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_nine_patch_stretch"), &TextureProgressBar::get_nine_patch_stretch);
 
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "fill_mode", PROPERTY_HINT_ENUM, "Left to Right,Right to Left,Top to Bottom,Bottom to Top,Clockwise,Counter Clockwise,Bilinear (Left and Right),Bilinear (Top and Bottom),Clockwise and Counter Clockwise"), "set_fill_mode", "get_fill_mode");
-	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "nine_patch_stretch"), "set_nine_patch_stretch", "get_nine_patch_stretch");
+	ADD_GROUP("Radial Fill", "radial_");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "radial_initial_angle", PROPERTY_HINT_RANGE, "0.0,360.0,0.1,slider,degrees"), "set_radial_initial_angle", "get_radial_initial_angle");
```

Comment:
The `slider` hint seems outdated, or is it supposed to be `hide_slider`?

Reply:
That hint isn't used anywhere else, so I've removed it. Other places that use `degrees` don't use `hide_slider` (regardless of whether it's an angle or velocity), so I've opted not to use it either.